scripts/editor.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# We have to use npm instead of yarn because it causes trouble in netlify
+
+# Link local mermaid to npm
+pushd packages/mermaid || exit
+npm link
+popd || exit
+
+# Clone the Mermaid Live Editor repository
+git clone https://github.com/mermaid-js/mermaid-live-editor.git
+
+# Change to the repository directory
+cd mermaid-live-editor || exit
```

Comment:
By the way, if you add a `set -euxo pipefail` to the front of your bash script, it will automatically exit the bash script if any command fails (so we don't need any `|| exit` bits)!

- `set -e`: Exits the bash script if any command fails.
- `set -x`: print out each line before running it (helping for debugging GitHub Actions, although I wouldn't do this in an users facing scripts)
- `set -u`: Exit the script trying to use unset variables, e.g. `$NON_EXISTANT_VARIABLE`.
- `set -o pipefail`: if any command in a pipe fails, make the whole pipe return an error code.

```suggestion
set -euxo pipefail

# We have to use npm instead of yarn because it causes trouble in netlify

# Link local mermaid to npm
pushd packages/mermaid
npm link
popd

# Clone the Mermaid Live Editor repository
git clone https://github.com/mermaid-js/mermaid-live-editor.git

# Change to the repository directory
cd mermaid-live-editor
```